engine/dbscan/src/main/java/org/opennms/alec/engine/dbscan/DBScanEngineFactory.java

```diff
@@ -72,7 +72,7 @@ public AbstractClusterEngine createEngine(MetricRegistry metrics) {
         if (factory.isPresent()) {
             return new DBScanEngine(metrics, epsilon, alpha, beta, factory.get());
         } else {
-            LOG.error("Wrong distance measure configuration {}, we'll use default {}", distanceMeasureFactoryName, alarmInSpaceAndTimeDistanceMeasureFactory.getName());
+            LOG.error("Warn distance measure configuration {}, we'll use default {}", distanceMeasureFactoryName, alarmInSpaceAndTimeDistanceMeasureFactory.getName());
```

Comment:
`Wrong` is more appropriate than `Warn` here.

Reply:
warn was intended for the log level, thank you to have spotted that